packages/client/src/StreamrClient.ts

```diff
@@ -168,8 +221,18 @@ export class StreamrClient {
     // --------------------------------------------------------------------------------------------
 
     /**
-     * Call last/from/range as appropriate based on arguments
+     * Performs a resend of stored historical data.
```

Comment:
Just "fetches"? At least to me the term "resend" is not very intuitive as long as resends are not subscriptions...